src/main/webapp/app/quiz/manage/manage-buttons/quiz-exercise-manage-buttons.component.html

```diff
@@ -1,36 +1,40 @@
 <div class="btn-group {{ isDetailPage ? 'flex-wrap' : 'flex-btn-group-container' }}">
     @if (quizExercise.isAtLeastTutor) {
         <div class="{{ isDetailPage ? 'd-flex' : 'btn-group-vertical me-1 mb-1' }} ">
-            <a id="preview-quiz" [routerLink]="[baseUrl, 'quiz-exercises', quizExercise.id, 'preview']" class="btn btn-success btn-sm me-1 mb-1">
```

Comment:
Look at this in the IDE, the diff is more complex than the actual changes 

Copied each button to its own diff (template / component out of scope for this quickfix) - so no two buttons in one column anymore